### DISCUSSION TITLE
Bootstrap 5 breaks on compilation via webpack

### DISCUSSION THREAD

**Discussion by ITspirit** [twbs/bootstrap#35425] [Q&A]
I´ve just implemented the whole stack via Webpack Encore. All modules are the newest available.

But when i integrate "@import '~bootstrap/scss/utilities/api';"

I receive this error:

`
SassError: $n: Invalid index 1 for a list with 0 elements.
  ╷
7 │   @if type-of($values) == "string" or type-of(nth($values, 1)) != "list" {
  │                                               ^^^^^^^^^^^^^^^
  ╵
  node_modules/bootstrap/scss/mixins/_utilities.scss 7:47    generate-utility()
  node_modules/bootstrap/scss/utilities/_api.scss 13:9       @content
  node_modules/bootstrap/scss/mixins/_breakpoints.scss 68:5  media-breakpoint-up()
  node_modules/bootstrap/scss/utilities/_api.scss 5:3        @import
`

Does anybody have had the same issue? Any ideas or recommendation ??

Regards,
Michael

**Comment by Assigyn** [comment]
Hello @ITspirit 

I've got the same problem since I've upgraded from 5.2.3 to 5.3.X. Did you find a solution ? 

Thanks,

**Reply by julien-deramond** [reply]
Hey @Assigyn 
Would it be possible to create a reproducible environment by using StackBlitz (or any other tools)?

**Reply by megabulk** [reply]
I have the same problem, fixed by changing the line to 
`  @if type-of($values) == "string" or (length($values) != 0 and type-of(nth($values, 1)) != "list") {
`
Does that look right? It seems to work for me.

**Comment by Fasalukm** [comment]
I'm also having this issue. Has anyone found a real fix?
